### PR TITLE
ui lint check yml

### PR DESCRIPTION
**Pull request by 0xmihirsahu**
added `ui-linting.yml` in `.github/workflows` for checking lint on every PR for the UI.

### CODE REVIEW

**Comment by 0xmihirsahu** [comment]
#142

File: .github/workflows/ui-linting.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: UI Lint Check on every PR
```
Comment (varex83) [inline .github/workflows/ui-linting.yml]:
```suggestion
name: [UI] Lint
```

File: .github/workflows/ui-linting.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: [UI] Lint
```
Comment (varex83) [inline .github/workflows/ui-linting.yml]:
```suggestion
name: 'UI: Lint'
```

**Comment by varex83** [comment]
> there seems to be issues. working on it

I managed to fix them already, LGTM now

**Review by varex83 — APPROVED** [review]
LGTM

**Comment by 0xmihirsahu** [comment]
> > there seems to be issues. working on it
> 
> I managed to fix them already, LGTM now

thanks :)

**Review by satyambnsal — APPROVED** [review]

**Pull request merged**